.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,3 +25,13 @@
 
 ### Functionality After
```

Comment:
Is it intentional to have the Functionality Before/After section only be part of the bug template ?

Reply:
@mohanraj-r  I think so. Before/After would only apply to Bug Fix.

Reply:
@mohanraj-r  I updated the template, could you take a look for the updates?